rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -159,6 +155,29 @@ public static boolean scriptCorrectlySpendsTx(BtcTransaction tx, int index, Scri
         }
     }
 
+    /**
+     * @param amountSent      amount sent
+     * @param activations     the network HF activations configuration
+     * @param bridgeConstants the Bridge constants
+     * @return true if amount sent is over the minimum for pegin transaction
+     */
+    public static boolean isAmountSentOverMinimum(Coin amountSent, ActivationConfig.ForBlock activations, BridgeConstants bridgeConstants) {
```

Comment:
It may be worth to change this in preparation for the validation per utxo.
Instead of expecting the amountSent you could receive the BtcTransaction and addresses.
This method would then combine the logic of `getAmountSentToAddresses` and `isAmountSentOverMinimum`. In a future story we could change HOW we get that amount

Reply:
It could be good. The only thing I see is that the `getAmountSentToAddresses` new method is in the other branch([the one for check](https://github.com/rsksmart/rskj/pull/1705)). It would be better to include here after merge that one in the integration branch. Anyway, I did a refactor considering what you said and using the method `getAmountSentToAdress` as it is in this branch, take a look and let me know what you think.